File: hackerrank/30_day_of_code/day_13_abstract_classes.py
class MyBook(Book):
    def __init__(self, t, a, p):
        super()
        self.title = t
        self.author = a
        self.price = p

    def display(self):
        print(
            f"Title: {self.title}\nAuthor: {self.author}\nPrice: {self.price}")

File: hackerrank/30_day_of_code/day_13_abstract_classes.js
class MyBook extends Book {
    constructor(title, author, price) {
        super();
        this.title = title;
        this.author = author;
        this.price = price;
    }
    display() {
        console.log(
            `Title: ${this.title}\nAuthor: ${this.author}\nPrice: ${this.price}`,
        );
    }
}
